examples/fork/fork.bal

```diff
@@ -0,0 +1,28 @@
+import ballerina/io;
+import ballerina/lang.runtime;
+
+public function main() {
+    execute(true);
+    runtime:sleep(1);
```

Comment:
Why do we need this?

Reply:
I tried to make the output always same. I will remove this.